### PR TITLE
Update build and install instructions

### PR DESCRIPTION
**Pull request by efargas**
More clarify notes IMHO

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,31 +22,41 @@ API Documentation: http://wjwwood.github.com/serial/doc/1.1.0/index.html
...
+git clone https://github.com/ros/catkin.git
+git clone https://gitlab.com/ColossusGroup/serial.git
+cd ..
+```
+
+### Compile & Install with catkin
+---------------------------------
+```Shell
+sudo su
+./src/catkin/bin/catkin_make install -DCMAKE_INSTALL_PREFIX=/usr/local
```
Comment (wjwwood) [inline README.md]: I don't think this is a good recommended practice. The resulting build space output will be owned by root and I'd prefer not to suggest people install things to the system by default, but rather is should noted as an option.

File: README.md [hunk 2]
```diff
@@ -22,31 +22,41 @@ API Documentation: http://wjwwood.github.com/serial/doc/1.1.0/index.html
...
+sudo ldconfig -v | grep libserial
```
Comment (wjwwood) [inline README.md]: I don't know why this is necessary, but it's not portable to OS X.

File: README.md [hunk 3]
```diff
@@ -67,4 +77,4 @@ John Harrison <ash.gti@gmail.com>
 
 ### Contact
 
-William Woodall <william@osrfoundation.org>
+William Woodall <william@osrfoundation.org>
```
Comment (wjwwood) [inline README.md]: I don't see why this line needs to be changed.

**Comment by wjwwood** [comment]
I think these are useful instructions, but I'm tempted to hold off until I make the catkin dependency optional (it's on my todo list).

**Comment by korken89** [comment]
I have added a pull request that makes catkin optional.

**Comment by wjwwood** [comment]
Can you please address my comments with changes or responses? Thanks.

**Comment by wjwwood** [comment]
I'm going to decline this since there hasn't been progress on it since I asked for a few changes, and because I'm going to make the next version CMake only, but leave this version with catkin.

**Pull request closed without merging**